bunsen/bunsen-core-r4/src/main/java/com/cerner/bunsen/definitions/r4/R4StructureDefinitions.java

```diff
@@ -620,7 +620,7 @@ private <T> T transform(DefinitionVisitor<T> visitor,
           .sorted()
           .collect(Collectors.toList());
 
-      return visitor.visitReference(rootName, referenceTypes, childElements);
+      return visitor.visitReference(parentElement.toString(), referenceTypes, childElements);
```

Comment:
This is a non-trivial change as I think we are changing the contract here a little bit so it's worth being documented. IIUC, previously, the first argument here (`elementName`) was always `"Reference"`; and now we expect this to be the path to the actual reference element, e.g., `"Identifier.assigner"`, right? If yes, this is probably something javadoc worthy in the interface [here](https://github.com/google/fhir-data-pipes/blob/16fcc255cef4d2708b9941a854e6c638b2533d45/bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/DefinitionVisitor.java#L54).

Reply:
added details